src/Codeception/Lib/Generator/Cest.php

```diff
@@ -31,14 +31,11 @@ public function tryToTest({{actor}} \$I)
 
```

Comment:
Template above could have `void` return types.

Reply:
I think adding `: void` to test functions just makes them less readable and doesn't really add any value.